### PR TITLE
Feature/front end initialisation

### PR DESCRIPTION
**Pull request by graham-willis-druid**
Front end basic setup

### CODE REVIEW

File: packages/gafl-webapp-service/src/test-utils.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { createServer, init, server } from './server.js'
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/test-utils.js]: Is this for unit tests only?  If so can we move to __tests__ folder in the gafl-webapp-service package root?

File: packages/gafl-webapp-service/src/server.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  })
+
+  /*
+   * Decorator to make access to the session cache available as
+   * simple setters and getters hiding the session key.
+   */
+  server.decorate('request', 'cache', cacheDecorator(sessionCookieName))
+
+  process.on('unhandledRejection', err => {
+    console.log(err)
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/server.js]: Should use console.error

File: packages/gafl-webapp-service/src/pages/date-of-birth/route.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+import pageRoute from '../../page-route.js'
+import Joi from '@hapi/joi'
+
+export default pageRoute(
+  'date-of-birth',
+  '/date-of-birth',
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/pages/date-of-birth/route.js]: Can we use /buy/date-of birth - need to keep URL's as consistent with the existing service as possible.

File: packages/gafl-webapp-service/src/pages/name/route.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+import pageRoute from '../../page-route.js'
+import Joi from '@hapi/joi'
+
+export default pageRoute(
+  'name',
+  '/name',
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/pages/name/route.js]: Can we use /buy/name - need to keep URL's as consistent with the existing service as possible.

File: packages/gafl-webapp-service/src/server.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    process.exit(1)
```
Comment (sgardnerdell-ea) [inline packages/gafl-webapp-service/src/server.js]: Shouldn't exit the process

File: packages/gafl-webapp-service/src/test-utils.js (hunk 1 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/test-utils.js]: i tried it - it thinks its a test and tries to run it

File: packages/gafl-webapp-service/src/server.js (hunk 2 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/server.js]: done

File: packages/gafl-webapp-service/src/test-utils.js (hunk 1 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/test-utils.js]: Test suite failed to run

    Your test suite must contain at least one test.

      at onResult (../../node_modules/@jest/core/build/TestScheduler.js:173:18)
          at async Promise.all (index 23)
      at async TestScheduler.scheduleTests (../../node_modules/@jest/core/build/TestScheduler.js:260:11)
      at async runJest (../../node_modules/@jest/core/build/runJest.js:414:19)
      at async _run (../../node_modules/@jest/core/build/cli/index.js:371:7)
      at async runCLI (../../node_modules/@jest/core/build/cli/index.js:224:3)

I have moved it to /tests

File: packages/gafl-webapp-service/src/server.js (hunk 5 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/server.js]: done

File: packages/gafl-webapp-service/src/pages/name/route.js (hunk 4 above)
Comment (graham-willis-druid) [reply packages/gafl-webapp-service/src/pages/name/route.js]: done

**Pull request merged**